### PR TITLE
Fixed some mistakes in documentation on formats page

### PR DESCRIPTION
**Pull request by bsekachev**
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

## Summary by CodeRabbit

- **Documentation**
  - Updated and clarified format definitions and associated tasks for `CVAT for images 1.1`, `CVAT for video 1.1`, `Datumaro 1.0`, `MOT 1.0`, and `MOTS PNG 1.0`.
  - Enhanced descriptions to specify tasks, models, and shapes more accurately.

### CODE REVIEW

File: site/content/en/docs/manual/advanced/formats/_index.md [hunk 1]
```diff
@@ -23,23 +23,23 @@ The table below outlines the available formats for data export in CVAT.
...
+| [MOT 1.0](format-mot)                                                                                                               | .txt          | Video Tracking, <br>Detection                               | SORT, MOT-Net, IOU Tracker, <br>and others.                                                                                                                                             | Bounding Boxes                                                                 | Specific attributes  | Supported     |
+| [MOTS PNG 1.0](format-mots)                                                                                                         | .png<br>.txt  | Video Tracking, <br>Detection                               | SORT, MOT-Net, IOU Tracker, <br>and others.                                                                                                                                             | Bounding Boxes, Masks                                                          | Specific attributes  | Supported     |
```
Comment (zhiltsov-max) [inline site/content/en/docs/manual/advanced/formats/_index.md:42]: These two formats are primarily supposed for tracks.

Comment (bsekachev) [reply site/content/en/docs/manual/advanced/formats/_index.md:42]: And?

Comment (zhiltsov-max) [reply site/content/en/docs/manual/advanced/formats/_index.md:42]: And the diff shows that Tracks were removed from the description.

Comment (bsekachev) [reply site/content/en/docs/manual/advanced/formats/_index.md:42]: We have a separated column for tracks

image

Also displaying "Tracks" in column with name: "Shapes" looks weird

Comment (zhiltsov-max) [reply site/content/en/docs/manual/advanced/formats/_index.md:42]: BTW, I also see a small discrepancy in the `Type` column, e.g. in `JSON` and `.xml`. It seems like it also should use a uniform format.

Comment (bsekachev) [reply site/content/en/docs/manual/advanced/formats/_index.md:42]: Agree, but I did not try to fix everything in this pr

@mdacoca

**Review by mdacoca — APPROVED** [review]

**Pull request merged**